src/components/BTable/BTable.md

```diff
@@ -0,0 +1,68 @@
+The headers array holds objects that represent columns
+
+| Key      | Required/Optional | Default | Description                              |
+| -------- | ----------------- | ------- | ---------------------------------------- |
+| name     | Required          |         | This is what will be displayed as the column header.  |
+| key      | Required          |         | This is the matching key for that column to access the corresponding data in the rows. |
+| sort     | Optional          |         | Enables sorting on that column.                   |
+| render   | Optional          |         | Holds the rendering function name if applicabled.                   |
+
+```js
+
+const makeLink = function(data) {
+  return `<a href=${ data.id }>${ data.name }</a>`
+}
+
+const makeImage = function(data) {
+  return `<img src="${ data.source }" alt="${ data.name }" width="30" height="30">`
+}
+
+const makeLinkGroup = function(data) {
+  let html = ''
+  data.forEach(function(action) {
+    switch(action) {
+      case 'delete':
+        html += `<span class="ion-ios-trash" style="color: #4d90fe; font-size: 18px"></span><a href=${ data.id } class='link-group'> Delete</a>`
+        break;
+      case 'edit':
+        html += `<span class="ion-edit" style="color: #4d90fe; font-size: 18px"></span><a href=${ data.id } class='link-group'> Edit</a>`
+        break;
+    }
+  })
+  return html
+}
+
+const headers = [
+  {name: 'Header 1', key: 'header_one', sort: true}, 
+  {name: 'Header 2', key: 'header_two', render: makeLink, sort: true},
```

Comment:
it's sortable, but all values are the same :(

Reply:
im not sure what you mean?  This just lets the column be sorted, the sorting will be handled by the end user.